backend_new/prisma/seed-helpers/listing-factory.ts

```diff
@@ -0,0 +1,388 @@
+  RegionEnum,
+  ApplicationMethodsTypeEnum,
+  ListingEventsTypeEnum,
+  MultiselectQuestionsApplicationSectionEnum,
+  UnitsStatusEnum,
+} from '@prisma/client';
+
+export const listingFactory = (
+  i: number,
+  jurisdictionId: string,
+): Prisma.ListingsCreateInput => ({
+  additionalApplicationSubmissionNotes: `additionalApplicationSubmissionNotes: ${i}`,
+  digitalApplication: true,
+  commonDigitalApplication: true,
+  paperApplication: false,
+  referralOpportunity: true,
+  assets: '',
+  accessibility: `accessibility: ${i}`,
+  amenities: `amenities: ${i}`,
+  buildingTotalUnits: i,
+  developer: `developer: ${i}`,
+  householdSizeMax: 1,
+  householdSizeMin: i,
+  neighborhood: `neighborhood: ${i}`,
+  petPolicy: `petPolicy: ${i}`,
+  smokingPolicy: `smokingPolicy: ${i}`,
+  unitsAvailable: i - 1,
+  unitAmenities: `unitAmenities: ${i}`,
+  servicesOffered: `servicesOffered: ${i}`,
+  yearBuilt: 2000 + i,
+  applicationDueDate: new Date(),
+  applicationOpenDate: new Date(),
+  applicationFee: `applicationFee: ${i}`,
+  applicationOrganization: `applicationOrganization: ${i}`,
+  applicationPickUpAddressOfficeHours: `applicationPickUpAddressOfficeHours: ${i}`,
+  applicationPickUpAddressType: ApplicationAddressTypeEnum.leasingAgent,
+  applicationDropOffAddressOfficeHours: `applicationDropOffAddressOfficeHours: ${i}`,
+  applicationDropOffAddressType: ApplicationAddressTypeEnum.leasingAgent,
+  applicationMailingAddressType: ApplicationAddressTypeEnum.leasingAgent,
+  buildingSelectionCriteria: `buildingSelectionCriteria: ${i}`,
+  costsNotIncluded: `costsNotIncluded: ${i}`,
+  creditHistory: `creditHistory: ${i}`,
+  criminalBackground: `criminalBackground: ${i}`,
+  depositMin: `depositMin: ${i}`,
+  depositMax: `depositMax: ${i}`,
+  depositHelperText: `depositHelperText: ${i}`,
+  disableUnitsAccordion: true,
+  leasingAgentEmail: `leasingAgentEmail: ${i}`,
+  leasingAgentName: `leasingAgentName: ${i}`,
+  leasingAgentOfficeHours: `leasingAgentOfficeHours: ${i}`,
+  leasingAgentPhone: `leasingAgentPhone: ${i}`,
+  leasingAgentTitle: `leasingAgentTitle: ${i}`,
+  name: `name: ${i}`,
+  postmarkedApplicationsReceivedByDate: new Date(),
+  programRules: `programRules: ${i}`,
+  rentalAssistance: `rentalAssistance: ${i}`,
+  rentalHistory: `rentalHistory: ${i}`,
+  requiredDocuments: `requiredDocuments: ${i}`,
+  specialNotes: `specialNotes: ${i}`,
+  waitlistCurrentSize: i,
+  waitlistMaxSize: i + 1,
+  whatToExpect: `whatToExpect: ${i}`,
+  status: ListingsStatusEnum.active,
+  reviewOrderType: ReviewOrderTypeEnum.firstComeFirstServe,
+  displayWaitlistSize: true,
+  reservedCommunityDescription: `reservedCommunityDescription: ${i}`,
+  reservedCommunityMinAge: i * 10,
+  resultLink: `resultLink: ${i}`,
+  isWaitlistOpen: true,
+  waitlistOpenSpots: i,
+  customMapPin: false,
+  publishedAt: new Date(),
+
+  closedAt: new Date(),
+  afsLastRunAt: null,
+  lastApplicationUpdateAt: new Date(),
+  listingsBuildingAddress: {
```

Comment:
For the addresses, I'm wondering if we should make it to be more realistic.  Once we connect this to the front-end we will probably want addresses we can validate. As is this long/lat is in the middle of the ocean

Reply:
If this is just for testing that makes sense, but if this is seed data for starting up for dev I +1 Morgan